docs/ide/refactoring/change-signature.md

```diff
@@ -2,6 +2,7 @@
 description: "Learn more about: Change Signature"
 title: "Change Signature"
 ms.date: "09/18/2022"
+f1_keywords: ["vc.pkg.refactoring.changesignaturedlg"]
```

Comment:
I'm less familiar with this area. Can you help me understand why these keywords are prefaced with VC Package?